spec/policies/pageflow/folder_policy_spec.rb

```diff
@@ -22,7 +22,7 @@ module Pageflow
     it 'does not permit admins when there is no more than one account' do
       Account.delete_all
       folder = create(:folder)
-      admin = create(:user, :admin)
+      admin = create(:user, :admin, on: folder.account)
```

Comment:
Why is this needed now? Given admins an account makes sense, but why does the folder policy have to change?